app/frontend/wallet/helpers/request.ts

```diff
@@ -14,8 +14,8 @@ const request = async function request(url, method = 'GET', body = null, headers
   }
 
   try {
-    const response = await fetch(url, requestParams)
-    if (!response) throw NamedError('NetworkError')
+    const response = await fetch(url, (requestParams as any)) // TS does not like credentials
```

Comment:
Figure out if we can convince TS that requestParams credentials is ok. (I suspect type is lost on object.assign above). Also, when we are at it, we can just assing to `requestParams` there directly,